Driverless AI/risk-assessment-tools-in-the-world-of-ai-the-social-sciences-and-humanities/risk-assessment-tools-in-the-world-of-ai-the-social-sciences-and-humanities.md

```diff
@@ -0,0 +1,475 @@
+
+![sensitivity-analysis-9-id-1-prior-count](assets/sensitivity-analysis-9-id-1-prior-count.jpg)
+
+Now let's look if decreasing a priors_count by one will change a prediction. Before moving forward, click the backtrack clock icon on the top right corner where the table is displayed. In prediction number 1 ([ID] 1), let's decrease the prios_count by one. Immediately rescore the graph. Something similar to the image above should appear. Here we see that the prediction was flipped and what is concerning is that the change is drastic. The prediction went from 0.45075 to 0.38016. Imagine that this model is being used, and this person is predicted to be true two_year_recid because of one extra prior count that was possibly wrongly added to the record. The just stated scenario is not hard to believe, given the judicial system's existing bias, especially when we arrest people on the streets.  
+
+![sensitivity-analysis-10-age-cat](assets/sensitivity-analysis-10-age-cat.jpg)
+
+As mentioned above, age was also another factor at the global level, and with that in mind, let's see if that's true for the local level. Click on the backtrack clock icon and modify the age_cat value for prediction number one. Change the age_cat value from Less than 25 to Greater than 45, after, rescore the graph. We see that the prediction is also flipped, which isn't good because we don't want age to determine a true or false two_year_recid. 
+
+With the above, we can conclude that at the local level, we also have a bias in the form of priors_count where priors_count is not a true reflection of someone's true criminal record if we consider the racial bias within the judicial system. We also see that ageism (a type of bias) is introduced to the model when we see that age_cat can determine a true or false two_year_recid. In conclusion, bias exists at the local level.  
+
+
+## Task 8: The Reciprocal Impact that Should Exist Between AI, the Social Sciences, and Humanities
+
+At this point, many will wonder how we can solve these issues presented at the global and local levels. A solution will be to have the humanities and social sciences integrated and extend their work to the process of AI model creation while the AI world makes it standard to consult the humanities and social sciences. To understand the reciprocal impact that should exist between AI, the social sciences, and humanities, we will explore five problems that can arise if we were to use our model or similar models made by others. Simultaneously, we will discuss how that reciprocal impact can lead to a more acceptable and fair integration of AI into our daily lives and the judicial system. 
+
+**Problem 1: Confrontation Clause**
+
+To start our analysis, consider the opening statement, Rebecca Wexler wrote in her New York Times article title When a Computer Program Keeps You in Jail:
+
+- “The criminal justice system is becoming automated. At every stage — from policing and investigations
```

Comment:
Do you mean to say "sentence" instead of "senten"?

Reply:
correct